### PR TITLE
Release/8.0.0

### PR DESCRIPTION
**Pull request by ngachung**
### Added
- **PODAAC-4328**
  - Modify the code to be compliant with cumulus 11 input/output new schema
### Security

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>gov.nasa.podaac</groupId>
   <artifactId>cumulus-metadata-aggregator</artifactId>
-  <version>7.0.1</version>
+  <version>8.0.0-alpha.5-SNAPSHOT</version>
```
Comment (jocelynmallon) [inline pom.xml:17]: Does GitHub actions update this automatically, or do we need to manually bump it to 8.0.0?

Comment (ngachung) [reply pom.xml:17]: It will automatically bump to 8.0.0 after we merge to master.

**Review by jocelynmallon — APPROVED** [review]
lgtm

File: pom.xml (hunk 1 above)
Comment (ngachung) [reply pom.xml:17]: Example https://github.com/podaac/cumulus-metadata-aggregator/commit/028ae187c02fd8d1c8fc4a143c7370dc3f8c32de though I wonder why the release branch didn't build to create an rc candidate.

**Pull request merged**